packages/@aws-cdk/aws-appmesh/test/test.gateway-route.ts

```diff
@@ -133,20 +133,36 @@ export = {
     },
   },
 
-  'Can export and import GatewayRoutes and perform actions'(test: Test) {
+  'Can import Gateway Routes using ARN and attributes'(test: Test) {
```

Comment:
Can you leave this test unchanged, and add a new one for `fromGatewayRouteAttributes` please?